README.md

```diff
@@ -162,17 +202,38 @@ The container will include scripts for your convenience to start and stop Jupyte
 /rapids/utils/stop_jupyter.sh
 ```
 
-### Docker Container with SIEM Integration
+#### Docker Container with SIEM Integration
 
-If you want a CLX container with SIEM integration (including data ingest), follow the steps above to build the CLX image. Then use `docker-compose` to start multiple containers running CLX, Kafka, and Zookeeper. 
+The following steps show how to use `docker-compose` to create a CLX environment ready for SIEM integration. We will be using `docker-compose` to start multiple containers running CLX, Kafka and Zookeeper.
 
-```aidl
+First, make sure to have the following installed:
+
+* [Docker Compose 1.27.4](https://docs.docker.com/compose/install/)
```

Comment:
It looks like Docker Compose now supports GPUs. Did we want to update this (and test) for v1.28? https://github.com/docker/compose/issues/6691#issuecomment-758460418

Reply:
ok that it's still a release candidate?
https://github.com/docker/compose/releases

Reply:
Looks like it, yeah good point. So you could just leave as-is then update when it's released. Or otherwise note that it's an option and RC.

Reply:
ok i'll update as soon as v1.28 is released. thx.